packages/server/src/utilities/rowProcessor/index.ts

```diff
@@ -134,8 +134,10 @@ async function processDefaultValues(table: Table, row: Row) {
 
   for (const [key, schema] of Object.entries(table.schema)) {
     if ("default" in schema && schema.default != null && row[key] == null) {
-      const processed = await processString(schema.default, ctx)
-
+      const processed =
+        typeof schema.default === "string"
+          ? await processString(schema.default, ctx)
+          : schema.default
```

Comment:
Default value is not always a string any more, as multi-select types have a default value that is an array. Therefore we only run binding enrichment now if it's a string.

Reply:
I'm not sure where we currently land on backend changes going to master vs going to this branch. It's messy to do any kind of split, we should commit to one or the other.

Reply:
Ah yes, good point. It's a nightmare developing this way, but I'll split this apart and do a backend PR to master first, then leave this as frontend only.

Reply:
Done, should be a clean split now.